lib/raidenclient/RaidenClient.ts

```diff
@@ -309,6 +281,24 @@ class RaidenClient extends BaseClient {
     const res = await this.sendRequest(endpoint, 'PATCH', { state: 'settled' });
   }
 
+  /**
+   * Send a payment.
```

Comment:
Minor note but I've been trying to follow the present-tense for method comment blocks like "Sends a payment" rather than "Send a payment."